juriscraper/opinions/united_states/state/conn.py

```diff
@@ -2,47 +2,46 @@
 CourtID: conn
 Court Short Name: Conn.
 Author: Asadullah Baig <asadullahbeg@outlook.com>
-
 History:
- - 2014-07-11: created
- - 2014-08-08, mlr: updated to fix InsanityError on case_dates
- - 2014-09-18, mlr: updated XPath to fix InsanityError on docket_numbers
- - 2015-06-17, mlr: made it more lenient about date formatting
- - 2016-07-21, arderyp: fixed to handle altered site format
- - 2017-01-10, arderyp: restructured to handle new format use case that includes
+- 2014-07-11: created
+- 2014-08-08, mlr: updated to fix InsanityError on case_dates
+- 2014-09-18, mlr: updated XPath to fix InsanityError on docket_numbers
+- 2015-06-17, mlr: made it more lenient about date formatting
+- 2016-07-21, arderyp: fixed to handle altered site format
+- 2017-01-10, arderyp: restructured to handle new format use case that includes
         opinions without dates and flagged for 'future' publication
+- 2022-02-02, satsuki-chan: Fixed docket and name separator, changed super
```

Comment:
I think we historically tab these over.  Was there a reason you deleted the spacing?

Reply:
These comments were tabed with just one space and the the black format configured in `pre-commit` didn't complain when I remove it.
Should I add the 4 spaces for the tab before this list of comments?

Reply:
I think the convention for history is to tab with 4.  But not for all the other comments.  You should be able to find a good example in the codebase.

Reply:
Add tab (4 spaces) for history logs.